UPGRADE.md

```diff
@@ -2,6 +2,17 @@
 
 ## 2.3
 
+### Changed constructor of multiple services to integrate them with the SuluEventLogBundle
```

Comment:
The SQL to create the new required table is missing in the UPGRADE.md
Also its missing that the `SuluEventLogBundle` is required to be registered in the `config/bundles.php` as discussed this bundle is not optional.

Reply:
done 👍